CBinding/Compiler/ICompiler.cs

```diff
@@ -30,47 +30,53 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
+using System;
+using System.Threading.Tasks;
+using System.IO;
+using System.Text;
+using System.Linq;
+
 using Mono.Addins;
 
 using MonoDevelop.Core;
 using MonoDevelop.Projects;
+using MonoDevelop.Core.Execution;
 
 namespace CBinding
 {
-	[TypeExtensionPoint ("/CBinding/Compilers")]
 	public interface ICompiler
```

Comment:
Is ICompiler still used by anything after the removal of the old project system?